### ISSUE TITLE
participles

### ISSUE THREAD

**Issue by olesar** [tanya-ling/middle-russian-parser#6]
Прекрасное рядом, в парадигмах вообще нет причастий. 
Добавляю файл participles.txt, который нам нужно обсудить.
@dmsichinava: как мы их трактуем, что с падежами?
@tanya-ling нужно размножить окончания на все парадигмы
https://github.com/olesar/middle-russian-parser/blob/master/parser%20files/addenda/participles.txt

**Comment by dmsichinava** [comment]
Но это же далеко не все, что бывает. По какому принципу отбирались именно
эти окончания? Почему нет -ным, -вши (ж. р.), -вше (мн. ч.), -вшим и т. п.?

А -щ- причастия, у которых И. ед. муж. -ый/-яй?



2017-10-30 21:04 GMT+03:00 Olga Lyashevskaya <notifications@github.com>:

**Comment by tanya-ling** [comment]
Да, причастий нет и не было никогда (кроме -л). Причастия это не так просто, как кажется - нужно лепить флексии прилагательных на основы глаголов. Для каждой глагольной парадигмы нужно понять номер основы (возможно, он всегда нулевой, а возможно, и не всегда, а в каких-то парадигмах может быть особый, а может и не быть, а, может, причастный суффикс рациональнее добавить к основе, чтобы флексии прилагательных оставить совсем без изменений). А, возможно, проще всего сделать их отдельной частью речи, чтобы не плодить парадигмы прилагательных (и ошибки в них).
В целом, это интересная задача, но довольно времязатратная. Мне как-то до сих пор не хватало мотивации за нее взяться.
